### PR TITLE
Miasma is now filtered through gas mask filters, at medium strength.

### PR DESCRIPTION
**Pull request by BurgerLUA**
## About The Pull Request

Miasma is now filtered through gas mask, at medium strength.

## Why It's Good For The Game

A little weird that miasma isn't filtered through gas masks despite that being something that'd make sense for gas masks to filter.

## Changelog

:cl: BurgerBB
balance: Miasma is now filtered through gas mask filters, at medium strength.
/:cl:

### CODE REVIEW

File: code/modules/clothing/masks/gas_filter.dm [hunk 1]
```diff
@@ -39,7 +39,8 @@
 		/datum/gas/nitrium,
 		/datum/gas/freon,
 		/datum/gas/hypernoblium,
-		/datum/gas/bz
+		/datum/gas/bz,
+		/datum/gas/miasma
```
Comment (Jolly-66) [inline code/modules/clothing/masks/gas_filter.dm:43]: Trailing comma? 🤔

Comment (BurgerLUA) [reply code/modules/clothing/masks/gas_filter.dm:43]: not sure what you mean

Comment (MrMelbert) [reply code/modules/clothing/masks/gas_filter.dm:43]:
```suggestion
		/datum/gas/miasma,
```
they mean this

Comment (BurgerLUA) [reply code/modules/clothing/masks/gas_filter.dm:43]: Is that required now per guidelines? Time to re-read the guidelines (last read 2019)

Comment (MrMelbert) [reply code/modules/clothing/masks/gas_filter.dm:43]: it helps people in the future avoid the error you made in the first commit, so we do it where possible

Comment (BurgerLUA) [reply code/modules/clothing/masks/gas_filter.dm]: The funny error I made in the first commit is me accidentally adding a ``.`` instead of a ``,``

Comment (MrMelbert) [reply code/modules/clothing/masks/gas_filter.dm]: yes, but if it had a trailing comma you wouldn't have to add a `,` in the first place

**Review by Ghommie — APPROVED** [review]

**Pull request merged**